File: src/main/java/com/store/dto/OrderItemResponseDto.java
package com.store.dto;

import com.store.domain.orders.OrderItemStatus;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class OrderItemResponseDto {

    private Long orderItemId;

    private OrderItemStatus status;

    private int quantity;

    private int price;

    private String color;

    private String size;

    private String imageUrl;

}

File: src/main/java/com/store/dto/OrderItemResponseDto.kt
package com.store.dto

import com.store.domain.orders.OrderItemStatus
import java.math.BigDecimal

data class OrderItemResponseDto(
    val orderItemId: Long? = null,
    val status: OrderItemStatus? = null,
    val quantity: Int = 0,
    val price: BigDecimal = BigDecimal.ZERO,
    val color: String? = null,
    val size: String? = null,
    val imageUrl: String? = null,
)
